datafusion/expr-common/src/signature.rs

```diff
@@ -135,6 +134,8 @@ pub enum TypeSignature {
     /// Null is considerd as `Utf8` by default
     /// Dictionary with string value type is also handled.
     String(usize),
+    /// Zero argument
+    ZeroArg,
```

Comment:
Maybe NoArgs would be more explanatory? 🤔

Reply:
I don't see the difference 👀

Reply:
Its choice of wording yes. Another option we can take from https://en.wikipedia.org/wiki/Arity and name it Nullary to be in sync with unary, binary functions we already use.